### PR TITLE
Mirroring and installation updates

### PR DESCRIPTION
**Pull request by tpitzen**

### CODE REVIEW

**Review by dlangst — APPROVED** [review]
Looks good to me.

File: src/pages/installing/using-image-mirroring.md [hunk 1]
```diff
@@ -6,14 +6,15 @@ slug: using-image-mirroring
...
+Images for IBM Watson Annotator for Clinical Data (ACD) are available from the IBM Entitled Registry through December, 2022. Images for Merative Annotator for Clinical Data are available from the ACD registry. ACD consumers need to migrate from IBM ACD to Merative ACD by the end of December, 2022. To help ensure all ACD images are available during this migration period, we have mirrored recent [versions](https://github.com/merative/acd-containers/blob/master/CHANGELOG.md#releases) of the IBM ACD images from the IBM Entitled Registry to the ACD registry. Credentials for the ACD registry can be used to pull either Merative or IBM ACD images, once mirroring is configured for the IBM ACD images.
```
Comment (herbeck558) [inline src/pages/installing/using-image-mirroring.md:17]: from the Merative ACD registry. ...

And if they use the mirror they don't need to migrate to Merative ACD by end of Dec 2022 - So maybe say something like "In order to allow existing consumers of IBM ACD time to migrate, the most recent versions of IBM ACD images [link] have been mirrored to the Merative ACD registry.  ACD consumers using IBM Watson Annotator for Clinical Data need to either migrate to the Merative ACD or setup mirroring to pull the IBM ACD images from the Merative ACD registry by end of 2022.

File: src/pages/installing/using-image-mirroring.md [hunk 2]
```diff
@@ -6,14 +6,15 @@ slug: using-image-mirroring
 toc: true
 ---
 
-Setting up container registry repository mirroring enables you to do the following:
+Image mirroring is a technique to pull a container image from one or more registries using the same fully qualified image name.  Setting up container registry repository mirroring enables you to do the following:
 
 - Configure your OpenShift Container Platform cluster to redirect requests to pull images from a repository on a source image registry and have it resolved by a repository on a mirrored image registry.
 - Identify multiple mirrored repositories for each target repository, to make sure that if one mirror is down, another can be used.
+- Enable IBM ACD Container Edition to pull images from the new Azure ACD image registry so IBM ACD will continue to run after December 31, 2022.
```
Comment (herbeck558) [inline src/pages/installing/using-image-mirroring.md:13]: from the new Merative ACD image registry (vs Azure )

**Review by herbeck558 — CHANGES_REQUESTED** [review]
couple comments - one more is do we have to say anything about IBM ROKS not supporting image policy/mirroring - i think they have that documented in their docs we could find to point too? @cagrev

File: src/pages/installing/using-image-mirroring.md (hunk 2 above)
Comment (tpitzen) [reply src/pages/installing/using-image-mirroring.md]: Fixed

File: src/pages/installing/using-image-mirroring.md (hunk 1 above)
Comment (tpitzen) [reply src/pages/installing/using-image-mirroring.md]: Fixed

**Review by herbeck558 — APPROVED** [review]
LTGM

**Pull request merged**